<launch>
    <param name ="/wheel_ratio_fl"  value="1"/>
    <param name ="/wheel_ratio_fr"  value="1"/>
    <param name ="/wheel_ratio_bl"  value="1"/>
    <param name ="/wheel_ratio_br"  value="1"/>

    <node name="udp_motorCtrl_node" pkg="wheel_tokyo_weili" type="udp_motorCtrl_node" output="screen"/>

    <node name="encoderCounter_node" pkg="wheel_tokyo_weili" type="encoderCounter_node" output="screen"/>

<!-- Launch 2 Arduino boards -->

    <node
        ns="arduino_motor"
        pkg="rosserial_python"
        type="serial_node.py"
        name="Arduino_motor_node">
    <param name = "port" value = "/dev/ttyACM0"/>
    <param name = "baud" value = "57600"/>
    </node>

    <node
        ns="arduino_encoder"
        pkg="rosserial_python"
        type="serial_node.py"
        name="Arduino_encoder_node">
    <param name = "port" value = "/dev/ttyACM1"/>
    <param name = "baud" value = "57600"/>
    </node>

</launch>

